websites/api-docs-smoke-test/src/@commercetools-docs/gatsby-theme-docs/overrides/site-icon.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import SiteIcon from '../../../icons/HttpApiSmall';
```

Comment:
The standard icons need to be provided by the ui kit so we can centrally change the icon, but the mapping to the microsite is still local.